src/Illuminate/Auth/Console/stubs/make/controllers/HomeController.stub

```diff
@@ -14,7 +14,6 @@ class HomeController extends Controller
      */
     public function __construct()
     {
-        $this->middleware('auth');
```

Comment:
I don't understand why you can't just leave it? It's just the `auth` middleware.